src/Lykke.Service.BlockchainSettings/Startup.cs

```diff
@@ -1,178 +1,69 @@
-
-                await ApplicationContainer.Resolve<IStartupManager>().StartAsync();
-
-                await Log.WriteMonitorAsync("", $"Env: {Program.EnvInfo}", "Started");
-
-                await AutoRegistrationInMonitoring.RegisterAsync(Configuration, _monitoringServiceUrl, Log);
-            }
-            catch (Exception ex)
+            return services.BuildServiceProvider<AppSettings>(options =>
             {
-                await Log.WriteFatalErrorAsync(nameof(Startup), nameof(StartApplication), "", ex);
-                throw;
-            }
-        }
+                options.SwaggerOptions = _swaggerOptions;
 
-        protected override async Task StopApplication()
-        {
-            try
-            {
-                // NOTE: Service still can recieve and process requests here, so take care about it if you add logic here.
-
-                await ApplicationContainer.Resolve<IShutdownManager>().StopAsync();
-            }
-            catch (Exception ex)
-            {
-                if (Log != null)
+                options.Swagger = genOptions =>
                 {
-                    await Log.WriteFatalErrorAsync(nameof(Startup), nameof(StopApplication), "", ex);
-                }
-                throw;
-            }
-        }
-
-        protected override async Task CleanUp()
-        {
-            try
-            {
-                // NOTE: Service can't recieve and process requests here, so you can destroy all resources
+                    genOptions.OperationFilter<ApiKeyHeaderAccessTokenOperationFilter>();
+                };
 
-                if (Log != null)
+                options.ConfigureMvcOptions = mvcOptions =>
                 {
-                    await Log.WriteMonitorAsync("", $"Env: {Program.EnvInfo}", "Terminating");
-                }
+                    mvcOptions.Filters.Add(typeof(CheckModelStateAttribute), 0);
+                    mvcOptions.Filters.Add(typeof(ValidateActionParametersAttribute), 1);
+                };
 
-                ApplicationContainer.Dispose();
-            }
-            catch (Exception ex)
-            {
-                if (Log != null)
+                options.Logs = logs =>
                 {
-                    await Log.WriteFatalErrorAsync(nameof(Startup), nameof(CleanUp), "", ex);
-                    (Log as IDisposable)?.Dispose();
-                }
-                throw;
-            }
+                    logs.AzureTableName = "BlockchainSettingsLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.BlockchainSettingsService.Db.LogsConnectionString;
+                    
+                    logs.Extended = extendedLogs =>
+                    {
+                        extendedLogs.AddAdditionalSlackChannel("CommonBlockChainIntegration", channelOptions =>
+                        {
+                            channelOptions.MinLogLevel = Microsoft.Extensions.Logging.LogLevel.Information;
```

Comment:
Add spam guard disabling and notification messages enabling for both additional channels